test/utils/svm/validator.setup.ts

```diff
@@ -0,0 +1,37 @@
+import { spawn } from "child_process";
+import path from "node:path";
+import fs from "node:fs/promises";
+import { SvmSpokeClient } from "@across-protocol/contracts";
+
+const LEDGER_DIR = path.resolve(__dirname, "..", ".ledger");
+
+export default async () => {
+  // Always start with a clean ledger
+  await fs.rm(LEDGER_DIR, { recursive: true, force: true });
+
+  const args = [
+    "--clone-upgradeable-program",
```

Comment:
Currently, we’re cloning the program from Solana mainnet, which requires querying an RPC and introduces associated latency (~10-15 seconds). This will be replaced by the SVM Spoke binaries once they’re exported from the contracts repo.